### PR TITLE
Corrected import order of predicates.go.

### PR DESCRIPTION
**Pull request by k82cn**
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes # N/A

```release-note
None
```

### CODE REVIEW

File: plugin/pkg/scheduler/algorithm/predicates/predicates.go [hunk 1]
```diff
@@ -42,6 +41,8 @@ import (
 	"k8s.io/kubernetes/plugin/pkg/scheduler/schedulercache"
 	schedutil "k8s.io/kubernetes/plugin/pkg/scheduler/util"
 	"k8s.io/metrics/pkg/client/clientset_generated/clientset"
+
+	"github.com/golang/glog"
```
Comment (timothysc) [inline plugin/pkg/scheduler/algorithm/predicates/predicates.go]: This isn't really doing much and TBH the codebase is inconsistent on it's ordering.

Comment (k82cn) [inline plugin/pkg/scheduler/algorithm/predicates/predicates.go]: The order in my mind is "system lab", "k8s lab" and "3rd part lab" (split by empty line).

Comment (timothysc) [inline plugin/pkg/scheduler/algorithm/predicates/predicates.go]: Sure, but it's not consistent in the codebase.  e.g. why spend cycles on this if we don't have a standard.

Comment (k82cn) [inline plugin/pkg/scheduler/algorithm/predicates/predicates.go]: Got your point, that makes sense :).

**Pull request closed without merging**